pkg/utils/release/controller.go

```diff
@@ -243,3 +290,35 @@ func (h *SuiteController) DeleteAllSnapshotsInASpecificNamespace(namespace strin
 		return len(snapshotList.Items) == 0, nil
 	}, timeout)
 }
+
+// CreateComponentWithDockerSource create an has component from a given name, namespace, application, devfile and a container image
+func (h *SuiteController) CreateComponentWithDockerSource(applicationName, componentName, namespace, gitSourceURL, containerImageSource, outputContainerImage, secret string) (*appstudioApi.Component, error) {
```

Comment:
why is `SuiteController` h here instead of s like other places?

Reply:
you are right, 
fixed